modules/update-network-sysctl-allowlist.adoc

```diff
@@ -0,0 +1,201 @@
+    ^net.ipv6.conf.IFNAME.arp_accept$
+    ^net.ipv6.conf.IFNAME.arp_notify$
+    ^net.ipv6.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.retrans_time_ms$
+kind: ConfigMap
+metadata:
+  annotations:
+    kubernetes.io/description: |
+      Sysctl allowlist for nodes.
+    release.openshift.io/version: 4.12.0-0.nightly-2022-11-16-003434
+  creationTimestamp: "2022-11-17T14:09:27Z"
+  name: cni-sysctl-allowlist
+  namespace: openshift-multus
+  resourceVersion: "2422"
+  uid: 96d138a3-160e-4943-90ff-6108fa7c50c3
+----
+
+. Edit the list by using the following command:
++
+[source,terminal]
+----
+$ oc edit cm -n openshift-multus cni-sysctl-allowlist -oyaml
+----
++
+For example, to allow you to be able to implement stricter reverse path forwarding you need to add `^net.ipv4.conf.IFNAME.rp_filter$` and `^net.ipv6.conf.IFNAME.rp_filter$` to the list as shown here:
++
+[source,terminal]
+----
+# Please edit the object below. Lines beginning with a '#' will be ignored,
+# and an empty file will abort the edit. If an error occurs while saving this file will be
+# reopened with the relevant failures.
+#
+apiVersion: v1
+data:
+  allowlist.conf: |-
+    ^net.ipv4.conf.IFNAME.accept_redirects$
+    ^net.ipv4.conf.IFNAME.accept_source_route$
+    ^net.ipv4.conf.IFNAME.arp_accept$
+    ^net.ipv4.conf.IFNAME.arp_notify$
+    ^net.ipv4.conf.IFNAME.disable_policy$
+    ^net.ipv4.conf.IFNAME.secure_redirects$
+    ^net.ipv4.conf.IFNAME.send_redirects$
+    ^net.ipv4.conf.IFNAME.rp_filter$
+    ^net.ipv6.conf.IFNAME.accept_ra$
+    ^net.ipv6.conf.IFNAME.accept_redirects$
+    ^net.ipv6.conf.IFNAME.accept_source_route$
+    ^net.ipv6.conf.IFNAME.arp_accept$
+    ^net.ipv6.conf.IFNAME.arp_notify$
+    ^net.ipv6.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.retrans_time_ms$
+    ^net.ipv6.conf.IFNAME.rp_filter$
+----
+
+. Save the changes to the file and exit.
++
+[NOTE]
+====
+The removal of `sysctls` is also supported. Edit the file, remove the sysctl then save the changes and exit.
+====
+
+.Verification
+
+Follow these steps to verify that the updated `sysctl` list is now available on all nodes.
+
+. List all nodes in your cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
+
+. Enter into a debug session on one of the worker nodes. This step instantiates a debug pod called `<node_name>-debug`:
++
+[source,terminal]
+----
+$ oc debug node/<node_name>
+----
+
+. Set `/host` as the root directory within the debug shell. The debug pod mounts the root file system of the host in `/host` within the pod. By changing the root directory to `/host`, you can run binaries from the executable paths of the host:
++
+[source,terminal]
+----
+# chroot /host
+----
+
+. Change directory to `/etc/cni/tuning`:
++
+[source,terminal]
+----
+# cd /etc/cni/tuning
+----
+
+. List the content of the `allowlist.conf` file by using the following command:
++
+[source,terminal]
+----
+# cat allowlist.conf
+----
++
```

Comment:
I don't think we should be telling the user to look at files on the nodes.
It should be enough for the NetworkAttachmentDefinition described below to work on pods.

Describing this file could give the user the idea to modify this file manually, which would be overwritten whenever the CM is updated.

Reply:
I added this check as a verification, as a user I'd like to know if the changes I have made are propagated, but I am will to defer to others and remove it. @cgoncalves  wdyt on this leave it in or remove ?

Reply:
I tend to agree with @mmirecki. However, I understand the need to document verification steps.
An effective way to verify would be by creating a pod with the two added sysctls (`net.ipv4.conf.IFNAME.rp_filter` and `net.ipv6.conf.IFNAME.rp_filter`) and verifying that it was created successfully.

Reply:
OK @cgoncalves I have that underneath with link to section describing how to do that. Who is best to QE this section ? I think I will rewrite this piece based on that

Reply:
See the QE assignee field at https://issues.redhat.com/browse/CNF-5109

Reply:
@cgoncalves @mmirecki @kononovn I think this is a better verification step as you state. Hopefully all good now and can get through into docs. I need LGTM from QE @kononovn.